hack/e2e/fixtures/seed.yaml

```diff
@@ -84,7 +85,7 @@ spec:
       spec:
         gcp:
           region: europe-west3
-          zone_suffixes:
+          zoneSuffixes:
```

Comment:
It's also part of https://github.com/kubermatic/dashboard/blob/master/src/app/shared/entity/datacenter.ts.

Reply:
What are the files like datacenter.ts actually based on? The REST API in KKP still has the exact same types. Unless you use the CRDs 1:1 in the dashboard, I am not sure that the entity files need to be changed here.

Reply:
Ah, okay. So it's not updated in the API. Then ignore this comment.

Reply:
Yeah we only changed the CRDs but kept the API _mostly_ the same. There are a few changes like https://github.com/kubermatic/dashboard/pull/4131/commits/dab8d7823e55d87326e3efa63908b758c7d3fad5 (and this one needs a bit of you guys' love, cause I don't know how to properly handle a string[] for the Presets). ^^